versions.properties

```diff
@@ -1,12 +1,12 @@
-jacksonVersion=2.8.9
+jacksonVersion=2.9.8
```

Comment:
This is the cause of the current test failures when api-checker is being used. api-checker has a dependency on Jackson that requires Jackson >= 2.8.0 and < 2.9.0.

Updating api-checker to use the newer version of Jackson right now would put a blocker on this work, and I'm not certain how much effort it would  take to do so. It seems like we may want to upgrade to `2.8.10` and stay on that version for the time being. Of course, there  are known security vulnerabilities with that version, so I'll look into those and try to figure out how they affect us.

Reply:
Of course, we could also upgrade the version of Jackson we use directly and let the `extensions-filter-bundle` continue using the older working version for the time being.

Reply:
I like isolating our exposure as much as possible if we can't eliminate it completely.
Then we add a story to figure out how to move it in the external dependencies.

Reply:
I like isolating our exposure as much as possible if we can't eliminate it completely.
Then we add a story to figure out how to move it in the external dependencies.

Reply:
Gah. Unfortunately, that might not actually work. I can force the dependency back to the version that works with api-checker which will put the expected version of the Jackson JAR in the  `extensions-filter-bundle` EAR, however we still encounter a failure at runtime. 

My theory is that the thread classloader is being used rather than the EAR classloader. This would be because we do not load the Jackson class until we validate a request.

There should be a way to force class loading to occur at the appropriate time, but such a solution quickly pushes us into the realm where complexity outweighs the benefit.